Zend/tests/attributes/033_not_serializable.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+#[NotSerializable]
+--FILE--
+<?php
+
+#[NotSerializable]
+class C {}
```

Comment:
There will need to be tests about extending a NonSerializable class, which also needs to determine if this works or not if the child class implements ``__serialize()``/``__unserialize()``

Reply:
Maybe it's better to make it final? Why would anyone extend this attribute class?

Reply:
The attribute class *is* `final`. This is about classes that have the attribute applied.

Reply:
Oops, sorry, my bad. Great.